client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -104,21 +102,18 @@ const EmailProvidersStackedComparison = ( {
 
 		dispatch(
 			recordTracksEvent( 'calypso_email_providers_billing_interval_toggle_click', {
+				current_route: currentRoute,
```

Comment:
Do you know if the current route contains the full url with the site slug and domain name, or a sanitized version with this information replaced with tokens? In the former case we indeed wouldn't be able to aggregate events based on that `current_route` property, so it wouldn't be super useful.

Reply:
I missed this too! We should remove this and then work out if we have a good way of generating the general/sanitised version.